### PR TITLE
Update abseil conanfile.py to remove "\r" character in CMake target f…

### PR DESCRIPTION
**Pull request by Sfletche132**
Closes https://github.com/conan-io/conan-center-index/issues/16083

Specify library name and version:  **lib/1.0**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
Do you have a reproducible example where it fails on Windows without this change? Because there is this code for 2 years in abseil recipe and it never failed on Windows in CI.

(moreover the logic should be build machine dependent instead of host machine)

**Comment by Sfletche132** [comment]
> Do you have a reproducible example where it fails on Windows without this change? Because there is this code for 2 years in abseil recipe and it never failed on Windows in CI.
> 
> (moreover the logic should be build machine dependent instead of host machine)

There's an open issue here: https://github.com/conan-io/conan-center-index/issues/16083

I am seeing the same on my machine with the replicated setup as per the issue.

**Comment by SpaceIm** [comment]
I see, it's for build executed from msys2. Can you reference in top message that this PR closes issue you mentioned so that it can be closed automagically?

File: recipes/abseil/all/conanfile.py [hunk 1]
```diff
@@ -145,6 +145,9 @@ def _load_components_from_cmake_target_file(self, absl_target_file_path):
 
         abs_target_content = load(self, absl_target_file_path)
 
+        if self.settings.os == "Windows":
```
Comment (SpaceIm) [inline recipes/abseil/all/conanfile.py:148]: you can remove this condition

**Comment by SpaceIm** [comment]
You don't have to merge master after each new commit in master branch.

File: recipes/abseil/all/conanfile.py [hunk 2]
```diff
@@ -145,6 +145,8 @@ def _load_components_from_cmake_target_file(self, absl_target_file_path):
 
         abs_target_content = load(self, absl_target_file_path)
 
+        abs_target_content = abs_target_content.replace("\r\n", "\n")
```
Comment (prince-chrismc) [inline recipes/abseil/all/conanfile.py:148]: Please a comment or link so it's obvious this is required since the next contributor might not know :)

**Review by SpaceIm — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**